ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/fieldTemplates/label.html

```diff
@@ -1,4 +1,7 @@
-<template data-sly-template.label="${@ componentId, labelValue, labelVisible, bemBlock}">
+<template data-sly-template.label="${@ componentId, labelValue, labelVisible, bemBlock, withId}">
     <label for="${componentId}" class="${bemBlock}__label"
-           data-sly-test="${labelValue && (labelVisible==null || labelVisible==true)}">${labelValue}</label>
+           data-sly-test="${!withId && labelValue && (labelVisible==null || labelVisible==true)}">${labelValue}</label>
+
+    <label id="${componentId}" class="${bemBlock}__label"
+           data-sly-test="${withId && labelValue && (labelVisible==null || labelVisible==true)}">${labelValue}</label>
```

Comment:
can we test add attributes depending on sly-test, instead of repeating the label element.